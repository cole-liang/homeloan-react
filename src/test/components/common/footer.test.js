import React from "react";
import { shallow } from "enzyme";
import Footer from "../../../components/common/footer";

describe("Footer Component Test", () => {
  it("should render Footer component", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
