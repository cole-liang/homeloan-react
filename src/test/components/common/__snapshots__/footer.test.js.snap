// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Test should render Footer component 1`] = `
<styled.footer>
  <Bootstrap(Row)
    className="footerAbove d-flex justify-content-center justify-content-md-between"
  >
    <Col
      as="div"
      className="footerAbout order-1 order-md-0 d-flex align-items-center justify-content-md-start justify-content-center noPaddingAndMargin"
      md={6}
      xs={12}
    >
      <Link
        to="/"
      >
        <img
          alt="Zinq Logo"
          className="companyBrand"
          src="http://www.zinqsolutions.com.au/wp-content/uploads/2018/12/zinq-comingsoon-logo2.png"
        />
      </Link>
      <ul
        className="footerNav"
      >
        <li>
          Who We Are
        </li>
        <li>
          Contact Us
        </li>
        <li>
          Careers
        </li>
      </ul>
    </Col>
    <Col
      as="div"
      className="d-flex order-0 order-md-1 justify-content-center justify-content-md-end noPaddingAndMargin"
      md={2}
      xs={12}
    >
      <ul
        className="footerSocial d-flex align-items-center noPaddingAndMargin"
      >
        <li>
          <i
            className="fa fa-twitter-square"
          />
        </li>
        <li>
          <i
            className="fa fa-facebook-square"
          />
        </li>
        <li>
          <i
            className="fa fa-google-plus-square"
          />
        </li>
        <li>
          <i
            className="fa fa-instagram"
          />
        </li>
      </ul>
    </Col>
  </Bootstrap(Row)>
  <Bootstrap(Row)
    className="footerBelow d-flex justify-content-between"
  >
    <Col
      as="div"
      className="d-flex justify-content-center justify-content-md-start noPaddingAndMargin"
      md={6}
      xs={12}
    >
      <span>
        Copyright 
        <i
          className="fa fa-copyright"
        />
         2019
         
        Zinq Solutions
         All Rights Reserved.
      </span>
    </Col>
    <Col
      as="div"
      className="d-flex justify-content-center justify-content-md-end noPaddingAndMargin"
      md={6}
      xs={12}
    >
      <ul
        className="legalList noPaddingAndMargin"
      >
        <li>
          Privacy Policy
          <span
            className="dot"
          />
        </li>
        <li>
          Terms of Use
          <span
            className="dot"
          />
        </li>
        <li>
          Site Map
        </li>
      </ul>
    </Col>
  </Bootstrap(Row)>
</styled.footer>
`;
